online_store/gt_sports.html

```diff
@@ -6,94 +6,114 @@
     <title>goodTeens sports online store</title>
   </head>
   <body>
-<h2 class="column" id="h2title">goodTeens - Sports</h2>
+<h2 class="column" id="h2title">goodTeens - Sports</h2> <p>
 <h2 class="column" id="h2title2">Online Store</h2>
-<br><br><br><br><br>
-<p></p>
+<br><br>
```

Comment:
you don't need this br tags.
If you want to split you h2 tags you can have a margin between them and first card or if they are kind of a header then you can wrap them in header tag with fixed height (like 80px or so).